_plugins/auto-anchor.rb

```diff
@@ -0,0 +1,29 @@
+## Automatically add id tags to list items that are formatted like one
```

Comment:
- use snakecase for the names of Ruby files
- add `# frozen_string_literal: true` magic comment to top of Ruby files (same for the Gemfile)